item-trade/packages/contracts/.env

```diff
@@ -17,12 +17,15 @@ RPC_URL="http://127.0.0.1:8545"
 CHAIN_ID=31337
 
 #SMART STORAGE UNIT ID
+#Copy this info from in game smart storage unit
 SSU_ID=11196282436511424955311689125460433858625928370621647559384988961918634545666
 
+# Its a prerequisite to have the item deposited into the SSU
+# Once deposited get the itemId from https://blockchain-gateway-nova.nursery.reitnorf.com/smartassemblies/<ssu_id>
```

Comment:
Needs clearer explanation - users should get the itemId from the corresponding item in the inventory.storageItems array of their SSU at that address.